ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -54,6 +54,67 @@ ethernet_interfaces:
     <...>
 ```
 
+## Removal Ansible tags from AVD roles
+
+Support for Ansible tags across AVD roles has been replaced with variables to control this execution of various parts of AVD.
+
+The previous implementation of Ansible tags was not consistent and mostly undocumented. It often lead to inconsistent behaviors depending
+when the tags were used with `--tags` vs. `--skip-tags`.
+
+AVD now supports a number of variables which allow detailed control of which areas of AVD should be executed.
+Variables are only added for the roles providing multiple optional functionalities.
+
+| Ansible Role | Variable | Default value | Description |
+| ------------ | -------- | ------------- | ----------- |
+| All | `avd_verify_requirements` | `true` | Verify Ansible and Python requirements. |
+| All | avd_create_directories | `true` | Create required output directories if not present. |
+| `arista.avd.eos_designs` | `avd_eos_designs_facts` | `true` | Generate `avd_switch_facts` which is a prerequisite for `eos_designs_structured_config` and `eos_designs_documentation`. |
+| `arista.avd.eos_designs` | `avd_eos_designs_debug` | `false` | Dump all vars and facts per device after generating `avd_switch_facts`. |
+| `arista.avd.eos_designs` | `avd_eos_designs_structured_config` | `true` | Generate structured configuration per device. |
+| `arista.avd.eos_designs` | `eos_designs_documentation.enable` | `true` | Generate fabric Markdown documentation. |
```

Comment:
Should we prefix with `avd_`
Also should we rename ``eos_designs_documentation.enable` to something else, perhaps`eos_designs_documentation.fabric`?